src/Focusable.ts

```diff
@@ -203,6 +207,10 @@ export class FocusableAPI implements Types.FocusableAPI {
             onUncontrolled,
         } = options;
 
+        const onElement = (
+            options as Types.FindIteratorProps | { onElement: undefined }
```

Comment:
_findElement is the basic internal method used by all. The callback is not optional from the iterator perspective, but from the generic methods that does it all, it's optional. If I do something like `if ((options as Types.FindIteratorProps).onElement) { ... }`, it will be a line of generated code. And here it's solved strictly on typings level and adds nothing extra to the generated code.